20241009.md

```diff
@@ -0,0 +1,48 @@
+# RISE RP005 QEMU weekly report 2024-10-02
+
+## Work completed since last report
+
+[SiFive function performance report](https://drive.google.com/file/d/1i6RLfZISECriwIesDVeZ9x1TjZHoOO5w).
+
+- WP2
+  - Added inlining of tb functions in do_vsetvl.
+  - Compacted loop of the first Embecosm patch to 8 byte per iteration.
+  - Complete SiFive library function set.
+    - **Complete**
+  - Continue to work on the separation of the large load/store patch.
+    - **Stopped**
+  - Single instruction benchmarking relevant to SiFive library functions.
+  - Continue to work on AArch64 comparison.
+    - **In progress**
+
+- Actions:
+  - Max: share patches applied to our benchmarking scripts.
+  - Max: run SPECCPU 2017 results again with VLEN=128 and LMUL=1 and enabled OpenMP.
+
+## Work planned for the coming week
+
+- WP2
+  - Measure impact of new patches.
+  - Single instruction benchmarking relevant to SiFive library functions.
+  - Continue to work on AArch64 comparison.
+  - Work on new optimizations for vector loads/stores.
+    - explore x86 intrinsics in the new version of the patches.
```

Comment:
Identify optimizations for those of the 14 SiFve string/memory functions where the current patch shows no improvement.